sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -304,6 +308,8 @@ def compute_table_name(row):
 NOTE: This job name template does not have backwards compatibility guarantees.
 """
 BQ_JOB_NAME_TEMPLATE = "beam_bq_job_{job_type}_{job_id}_{step_id}{random}"
+"""The number of shards per destination when writing via streaming inserts."""
+DEFAULT_SHARDS_PER_DESTINATION = 500
```

Comment:
Seems like this will conflict with https://github.com/apache/beam/pull/12485 ?

Reply:
yeah, this needs to fix conflicts with them.